src/app/component/Field/Field.component.js

```diff
@@ -391,6 +391,7 @@ export default class Field extends PureComponent {
               name={ name }
               value={ value }
               disabled={ isDisabled }
+              form={ form }
```

Comment:
You only added this to the `text` type. This is not correct. I still insist on input abstraction into its own component.